content/example_test.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package content_test gives examples code of functions in the content package.
+package content_test
```

Comment:
All examples in `content/example_test.go` should be merged to `registry/remote/example_test.go`. Ideally, no `registry` should be referenced in the `content` package.

Reply:
But this is only content_test, can't it reference `registry` either? If I move everything to the `registry` package then godoc won't render example for FetchAll.

Reply:
Moved to sceonario-based examples in remote package